src/main/java/mods/eln/node/NodeBase.kt

```diff
@@ -154,15 +154,17 @@ abstract class NodeBase {
             }
             if (Eln.thermometerElement.checkSameItemStack(equipped)) {
                 val str = thermoMeterString(side)
+                if(str === null) return true;
```

Comment:
Maybe display a message like "No readings", so a player won't get confused about why they can't meter the cable for heat?